### ISSUE TITLE
[exhentai][e-hentai] not working

### ISSUE THREAD

**Issue by nille02** [fmd-project-team/FMD#132]
Since some time now only the scaled images are downloadable but today i notices that the download is not working anymore. 

Im Logged in and view the Information and Chanperlist, but downloads are not starting.

**Comment by Tmp341** [comment]
Tested for myself, even after adding ``Cookies``, download starts, but download speed stucks at ``0Kbps`` and never finishes (probably will fail later).

**Comment by SDXC** [comment]
Does ExHentai have a Region Lock? I only see a sad panda image...

**Comment by nille02** [comment]
no, you need to edit your cookies to enter it. copy the cookies from e-hentai to the exhentai url and it works. in your browser.

**Comment by Tmp341** [comment]
- I did the standard workaround and put my cookies under ExHentai's `Advanced` settings.
-  I logged in from E-Hentai, then opened ExHentai. 
- Copy-pasted link.
- FMD found the page, i added it to downloads.
- Downloaded only first page, not others.

Here are the cookies i used:

> igneous=xxxxxxxxx;
> ipb_member_id=xxxxxxxx;
> ipb_pass_hash=xxxxxxxxxxxxxxxxxxxxxxxxxxx;
> sk=xxxxxxxxxxxxxxxxxxxxxxxxxx;
> yay=louder
>Mozilla/5.0 (Windows NT 6.1; rv:69.0) Gecko/20100101 Firefox/69.0


It was working before, so i think they've changed something.

**Comment by Hypertyrannical** [comment]
It seems when I try to download the manga list from ExHentai, it starts to download, but at the 751 byte mark (and always at that mark) it stops and throws a popup error:

![image](https://user-images.githubusercontent.com/52367494/74079966-05737a00-4a04-11ea-8f34-96ff13f451b2.png)

In the Accounts tab, it reports my ExHentai login as OK, so I tried to do what the post just above this one described with the cookies. It didn't seem to make a difference, unfortunately.

**Comment by Tmp341** [comment]
> 
> 
> It seems when I try to download the manga list from ExHentai, it starts to download, but at the 751 byte mark (and always at that mark) it stops and throws a popup error:
> 
> ![image](https://user-images.githubusercontent.com/52367494/74079966-05737a00-4a04-11ea-8f34-96ff13f451b2.png)
> 
> In the Accounts tab, it reports my ExHentai login as OK, so I tried to do what the post just above this one described with the cookies. It didn't seem to make a difference, unfortunately.

Because exhentai code is broken and we couldn't update mangalist, so no file on the server. until someone fix that, you can't update database, it is one hell of a database anyway, so just copy the link you want to download and try to start it. It was working before (haven't tried it for some time).

**Comment by Hypertyrannical** [comment]
> > It seems when I try to download the manga list from ExHentai, it starts to download, but at the 751 byte mark (and always at that mark) it stops and throws a popup error:
> > ![image](https://user-images.githubusercontent.com/52367494/74079966-05737a00-4a04-11ea-8f34-96ff13f451b2.png)
> > In the Accounts tab, it reports my ExHentai login as OK, so I tried to do what the post just above this one described with the cookies. It didn't seem to make a difference, unfortunately.
> 
> Because exhentai code is broken and we couldn't update mangalist, so no file on the server. Until someone fix that, you can't update database, it is one hell of a database anyway, so just copy the link you want to download and try to start it. It was working before (haven't tried it for some time).

So what you're saying is that there's nothing you can do on your end? That's a shame. It would be nice to have even an outdated copy... any chance you'd be willing to update your exhentai db to https://anonfile.com/ or something? If so I recommend this method: `curl -F "file=@ExHentai.db" https://api.anonfile.com/upload` (It spits out some json with a download link once it's done.)

**Comment by Tmp341** [comment]
> So what you're saying is that there's nothing you can do on your end? That's a shame. It would be nice to have even an outdated copy... any chance you'd be willing to update your exhentai db to https://anonfile.com/ or something? If so I recommend this method: `curl -F "file=@ExHentai.db" https://api.anonfile.com/upload` (It spits out some json with a download link once it's done.)

I don't have outdated database, sorry.
